tasks/frontEnd/src/App.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <AppProvider>
+    <Page title="Judge.Me Reviews">
+      <Layout>
+        <LayoutSection>
+          <Button @click="toggleCollapsibleActive" style="margin-bottom: 20px;">{{ formButtonMessage }}</Button>
+          <Collapsible :open="collapsibleActive" :class="{ collapsibleActive: collapsibleActive }">
+            <Card>
+              <ReviewForm @add-review="addReview" :key="formReset" />
+            </Card>
+          </Collapsible>
+          <Checkbox label="Unverified reviews only" v-model="unverifiedOnly" />
+          <Divider />
+          <EmptyState
+            v-if="!reviews || reviews.length < 1"
+            heading="No reviews found">
+            <Text as="p">Add a review above or reload the page</Text>
+          </EmptyState>
+          <ReviewCard
+            v-for="(review, index) in reviews"
+            :key="review.id"
+            :reviewObject="review"
+            :unverifiedOnly="unverifiedOnly"
+            @remove-review="removeReview(review.id, index)"
+            @verify-review="verifyReview(review.id, index)"
+          />
+        </LayoutSection>
+      </Layout>
+    </Page>
+  </AppProvider>
+</template>
+
+<script setup>
+import { computed, onMounted, ref } from 'vue';
+import ReviewCard from './components/ReviewCard.vue';
+import ReviewForm from './components/ReviewForm.vue';
+
+const reviews = ref(null);
+const collapsibleActive = ref(false);
+const formButtonMessage = ref('Add new review');
+const unverifiedOnly = ref(false);
+const formReset = ref(0);
+
+const toggleCollapsibleActive = () => {
+  collapsibleActive.value = !collapsibleActive.value;
+  collapsibleActive.value ? formButtonMessage.value = 'Hide form' : formButtonMessage.value = 'Add a new review';
```

Comment:
So instead of having two concerns, this could remove the assignment on line 46, and directly use it on line 40. 

E.g.
`formButtonMessage = ref(formButtonMessage.value ? 'Hide form' : 'Add new review');`

Reply:
I tried this (though presumably you meant `formButtonMessage = ref(collapsibleActive.value ? 'Hide form' : 'Add new review');`), but it didn't match expected behaviour. Agree that it could be refactored to be more efficient though.